service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/PopulateStandardDirectionsHandler.java

```diff
@@ -98,17 +81,16 @@ public void populateStandardDirections(PopulateStandardDirectionsEvent event) th
             caseDataContent);
     }
 
-    private Map<String, Object> populateStandardDirections(
-        @RequestBody CallbackRequest callbackrequest) throws IOException {
+    //TODO: move logic to separate service and unit test FPLA-1512
+    private Map<String, Object> populateStandardDirections(CallbackRequest callbackrequest) throws IOException {
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        OrderDefinition standardDirectionOrder = ordersLookupService.getStandardDirectionOrder();
+        LocalDateTime hearingStartDate = getFirstHearingStartDate(caseData.getHearingDetails());
 
-        List<Element<Direction>> directions = standardDirectionOrder.getDirections()
+        List<Element<Direction>> directions = ordersLookupService.getStandardDirectionOrder().getDirections()
```

Comment:
Is it accidental refactor or expected to be in scope of this one?

A bit surprised to see changes related to directions in the hearing date refactoring. But not massively fussed as it is exactly the same method.

Reply:
it was during implementing the getFirstHearing method here, wanted to clean up a little as well. Can revert, let me know

Reply:
Happy for the cleanup.